edx_adapt/tests/logic_tests.py

```diff
@@ -24,18 +46,21 @@ class BaseTestCase(unittest.TestCase):
     def setUpClass(cls):
         cls.skills = ['center', 'shape', 'spread', 'x axis', 'y axis', 'h to d', 'd to h', 'histogram', 'None']
         cls.course_id = COURSE_ID + id_generator(3)
-        course_setup.setup_course_in_edxadapt(cls.course_id)
-
         cls.headers = {'Content-type': 'application/json'}
         cls.app = adapt_api.app.test_client()
+        cls.params = {
+            'headers': cls.headers,
+            'skills': cls.skills,
+            'course_id': cls.course_id,
+        }
+        _setup_course_in_edxadapt(cls.app, **cls.params)
+
 
     @classmethod
     def tearDownClass(cls):
         # NOTE(idegtiarov) sqlite is too slow for using on server we will support only MongoDB
-        mclient = pymongo.MongoClient()['edx-adapt']
-        mclient.drop_collection(cls.course_id)
-        regex = re.compile('_student_')
-        mclient.Generic.remove({'key': {'$regex': regex}})
+        mclient = pymongo.MongoClient()
+        mclient.drop_database('edx-adapt')
```

Comment:
Yes, but then you have a dev db on a local machine.
I just don't want to add a code which can let some stupid error or typo ruin data. Even if it's only on a dev machine.

If there's a lot of hardcode needs to be rewritten to make this change it's ok to do it in a separate PR, but put a TODO note here.